scvi_in_R.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>not sure the last line does anything here, shouldn't it be scvi$settings$progress_bar_style = "tqdm"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>